### PR TITLE
Add JDBC support in schema-loading helm chart

### PR DESCRIPTION
**Pull request by scalar-boney**
* JDBC support in schema-loading helm chart

### CODE REVIEW

File: charts/schema-loading/templates/batchjob-schema-loading.yaml [hunk 1]
```diff
@@ -19,8 +19,6 @@ spec:
         image: "{{ .Values.schemaLoading.image.repository }}:{{ .Values.schemaLoading.image.version }}"
         imagePullPolicy: "{{ .Values.schemaLoading.image.pullPolicy }}"
         args:
-        - "-P"
```
Comment (miseyu) [inline charts/schema-loading/templates/batchjob-schema-loading.yaml:22]: I see that `ContactPort` is only for cassandra.

**Review by miseyu — APPROVED** [review]
Thank you！
LGTM

File: charts/schema-loading/templates/batchjob-schema-loading.yaml (hunk 1 above)
Comment (scalar-boney) [reply charts/schema-loading/templates/batchjob-schema-loading.yaml:22]: Yes, https://github.com/scalar-labs/scalardb/tree/master/schema-loader#run

**Review by feeblefakie — APPROVED** [review]
LGTM!
But, I think we should update the title since some of the changes are not from JDBC support.

File: charts/schema-loading/templates/batchjob-schema-loading.yaml (hunk 1 above)
Comment (feeblefakie) [reply charts/schema-loading/templates/batchjob-schema-loading.yaml:22]: Is this change related to the JDBC support?

**Comment by scalar-boney** [comment]
> LGTM! But, I think we should update the title since some of the changes are not from JDBC support.

Sorry for the inconvenience.

I have created a separate PR for changes outside of JDBC support https://github.com/scalar-labs/helm-charts/pull/48

File: charts/schema-loading/templates/batchjob-schema-loading.yaml (hunk 1 above)
Comment (scalar-boney) [reply charts/schema-loading/templates/batchjob-schema-loading.yaml]: Sorry for the inconvenience.

I have created a separate PR for changes outside of JDBC support

**Review by miseyu — APPROVED** [review]
Thank you so much!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**